luigi/interface.py

```diff
@@ -59,6 +77,8 @@ def setup_interface_logging(conf_file='', level_name='DEBUG'):
         stream_handler.setFormatter(formatter)
 
         logger.addHandler(stream_handler)
+    elif isinstance(conf_file, dict):
```

Comment:
If we actually do down this path. The variable should be renamed to something like `path_or_dict`.